package.json

```diff
@@ -58,7 +58,7 @@
     "@safe-global/protocol-kit": "^3.1.1",
     "@safe-global/safe-apps-sdk": "^9.1.0",
     "@safe-global/safe-deployments": "^1.36.0",
-    "@safe-global/safe-gateway-typescript-sdk": "^3.21.5",
+    "@safe-global/safe-gateway-typescript-sdk": "3.21.6",
```

Comment:
well, this was not intended, but maybe a good thing. 
Normally I install it with @latest, which would add ^ before the version number. But this time because 3.21.6 was missing from npm and I ended up with 3.21.5, I tried installing it with @3.21.6 to see if the version is really missing or not... And once it was available on npm we ended up with "3.21.6".

With that said - as you can see an update to the sdk could brake other people's PR's because of type incompatibilities, so maybe it is better to be explicit here and specify the exact version?